src/pages/workspace/WorkspaceMembersPage.js

```diff
@@ -1,9 +1,7 @@
 import React from 'react';
 import _ from 'underscore';
 import lodashGet from 'lodash/get';
-import {
-    View, FlatList, TouchableOpacity,
-} from 'react-native';
+import {View, FlatList, TouchableOpacity} from 'react-native';
```

Comment:
Can you remove any changes like this that are unnecessary please? I will review again once there are only essential changes. If you are using any auto-lint tools please disable them.

Reply:
Not too concerned with it / don't have any real opinion on it. I'd say it's fine, but not really needed.

Reply:
@marcaaron @0xmiroslav do i need to revert lint fixes?

Reply:
@marcaaron i fix this after run `npm run lint`

Reply:
@0xmiroslav i have fix it please check